src/test/java/seedu/anilist/model/ModelManagerTest.java

```diff
@@ -118,13 +118,13 @@ public void equals() throws ParseException {
         // same values -> returns true
         modelManager = new ModelManager(animeList, userPrefs);
         ModelManager modelManagerCopy = new ModelManager(animeList, userPrefs);
-        assertTrue(modelManager.equals(modelManagerCopy));
+        assertEquals(modelManager, modelManagerCopy);
 
         // same object -> returns true
-        assertTrue(modelManager.equals(modelManager));
+        assertEquals(modelManager, modelManager);
 
         // null -> returns false
-        assertFalse(modelManager.equals(null));
+        assertFalse(modelManager == null);
```

Comment:
maybe use assert not equals here

Reply:
actually I have changed all back to assertFalse() / assertTrue() as using assertEquals()/ assertNotEquals() will bring code coverage down.